vimd2h.py

```diff
@@ -119,8 +119,13 @@ def to_html(self, contents):
         out = [ ]
 
         inexample = 0
+        lineno = 0
         for line in RE_NEWLINE.split(contents):
+            lineno += 1
             line = line.rstrip('\r\n')
+            titleline = lineno == 1 and re.match(r'^\*\S+\*\s.*', line)
```

Comment:
For consistency, `r'^\*\S+\*\s.*'` should be defined as a string constant at the top.

Reply:
Yup, was waiting for your comments on #8

Reply:
Force-pushed a different version, incorporating PAT_VIM_REF as discussed in #7. New CSS class is tentatively `flh` (first line of help), chiefly to avoid confusion with other previous and present "headers" and "titles", and also because `.f` is currently un-alocated (so no search-and-replace surprises).

I'm currently using
```
.flh { font-weight: bold; font-size: 120%; display: block; }
```